src/components/app/data/hooks/useEnterpriseFeatures.js

```diff
@@ -5,13 +5,14 @@ export default function useEnterpriseFeatures(queryOptions = {}) {
   return useEnterpriseLearner({
     ...queryOptionsRest,
     select: (data) => {
+      const transformedData = data.enterpriseFeatures || data.transformed.enterpriseFeatures;
```

Comment:
Similar feedback here around how we might want to ensure `data` always has the same data structure, regardless of when derived from the BFF query or the fallback query.